virus/Blumevirus_macaceinfi.md

```diff
@@ -0,0 +1,90 @@
+
+The samples mentioned above are 27 out of the total 28 sample runs that had the sequence of B. macaceinfi.
+
+### Looking Into Potential Hosts
+The sample that has not yet been mentioned is a viral sample from Influenza A virus, bulked with additional monkey RNA. It is slightly different from the other samples as the sample is not directly sourced from the gut or rectum, instead, it is a Influenza A sample amongst the list of viruses included in the target enrichment panel of the study "Enrichment post-library preparation enhances the sensitivity of high-throughput sequencing-based detection and characterization of viruses from complex samples" [Paskey AC et al. 2019]. What relates it to the other sequenced runs we have identified is that monkey RNA was added to the sample, and the lab host used were Vero cells, a common cell line extracted from the kidney of an African green monkey [Ammerman NC et al. 2008]. This ties back to the common factor among samples, its relatedness to monkeys.
+
+Basedo on observations across all the samples with the presence of B. macaceinfi, there is a high chance that gut microbiota of these rhesus monkeys are a potential host. With that prediction, we also looked at Figure 3 in the publication associated with our index case, which shows the relative gene expression level across all 24 samples. We identified a few common types of bacteria in both the control and ICD treated samples, which are Prevotella, Bacteroides, Megasphaera, Clostridium, Ruminococcus, Eubacterium, and other less expressed bacteria [Westreich ST et al. 2019]. Further research on whether these bacteria are host cells of B. macaceinfi could be of significant impact.
+
+### A Close Relative of Leviviridae sp.
+Based on results generated using the blastx tool in NCBI, our novel virus is most closely related to the virus Leviviridae sp., with a percent identity of 44.54% to its RdRP protein based off a Third Party Annotation (TPA), and a percent identity of 42.66% to the whole RdRP protein. Leviviridae sp. also belongs to the family Blumeviridae, however, because B. macaceinfi and Leviviridae sp. are only related on the class level, it is not extremely significant in terms better understanding our novel virus. More details confirming the structure and particular features of B. macaceinfi are required to further categorize it into the most suitable genus.
+
+### Virus Genome
+
+![Genome Map](img/genomemap.png)
```

Comment:
Can you use a `<virus_name>` folder path for the images/data files please. Look at how the data is organized in the repository currently.

Reply:
Just added. Thanks.